components/kyma-environment-broker/internal/process/deprovisioning/remove_instance_step.go

```diff
@@ -103,3 +109,22 @@ func (s RemoveInstanceStep) removeInstancePermanently(instanceID string, log log
 
 	return 0
 }
+
+func (s RemoveInstanceStep) markInstanceNeedsRetrySomeSteps(instanceID string, log logrus.FieldLogger) time.Duration {
+	backoff := time.Second
+
+	instance, err := s.instanceStorage.GetByID(instanceID)
```

Comment:
if the instance is not existing (an edge case) - let's log a warning and leave the step without backoff